app/components/ui/ExpandableTransactionDetails.tsx

```diff
@@ -0,0 +1,159 @@
+import { Info, CheckCircle, XCircle } from "lucide-react";
+import { type MintTransaction } from "~/server/Mint/types";
+import { AnimatedHourglass } from "./AnimatedHourglass";
+import { useIndexerNetworkContext } from "~/providers/IndexerNetworkProvider";
+import { NumericFormat } from "react-number-format";
+import { formatBTC } from "~/lib/denoms";
+
+interface ExpandableTransactionDetailsProps {
+	transaction: MintTransaction;
+}
+
+export function ExpandableTransactionDetails({ transaction }: ExpandableTransactionDetailsProps) {
+	const { bitcoinConfig } = useIndexerNetworkContext();
+	const confirmationThreshold = bitcoinConfig?.confirmationThreshold || 3;
+	const blockTime = bitcoinConfig?.blockTime || 120;
+	const operationDate = new Date(transaction.operationStartDate || transaction.timestamp);
+
+	const estimatedTimeRemaining = () => {
+		if (transaction.status !== "confirming") return null;
+		const remainingConfirmations = Math.max(0, confirmationThreshold - transaction.numberOfConfirmation);
+		const estimatedMinutes = remainingConfirmations * (blockTime / 60);
+		return Math.ceil(estimatedMinutes);
+	};
+
+	const formatTimeRemaining = (minutes: number | null) => {
+		if (!minutes || minutes <= 0) return null;
+		if (minutes === 1) return "~1 minute";
+		return `~${minutes} minutes`;
+	};
+
+	const isBroadcasted = () => {
+		return transaction.status !== "confirming" || transaction.numberOfConfirmation > 0;
+	};
+
+	const isConfirmed = () => {
+		return transaction.numberOfConfirmation >= confirmationThreshold;
+	};
+
+	const isMinted = () => {
```

Comment:
you can just store them in a variable instead of returning boolean

const example = a===2